webknossos/webknossos/dataset/dataset.py

```diff
@@ -106,6 +109,33 @@ def _find_array_info(layer_path: Path) -> Optional[ArrayInfo]:
 )
 
 
+def _guess_if_segmentation_path(filepath: Path) -> bool:
+    return "segmentation" in str(filepath)
+
+
+def _has_image_z_dimension(
```

Comment:
These two functions don’t seem to be too specific to the Dataset class, maybe they could be moved to utils? This file is starting to get pretty long 😅